src/pages/profiles/ProfileUsedByDefaultProject.tsx

```diff
@@ -33,6 +40,13 @@ const ProfileUsedByDefaultProject: FC<Props> = ({
             )}
             {project.instances.length > 0 && (
               <ExpandableList
+                customShowAll={() => {
```

Comment:
We probably shouldn't use the Expandable list here, otherwise the experience of "show all" is inconsistent.

Also, I think the link to the filtered instance list is useful even with 2 instances to bulk-stop or restart them.

Reply:
Hmm, yes, could be, but what would be the best way to represent it visually then? @al-bakalova